metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -1348,3 +1349,36 @@ def visit_join_to_time_spine_node(self, node: JoinToTimeSpineNode) -> SqlDataSet
                 where=where,
             ),
         )
+
+    def visit_min_max_node(self, node: MinMaxNode) -> SqlDataSet:  # noqa: D
+        parent_data_set = node.parent_node.accept(self)
+        parent_alias = self._next_unique_table_alias()
+
+        assert (
+            len(parent_data_set.sql_select_node.select_columns) == 1
+        ), "MinMaxNode supports exactly one parent select column."
+        parent_column_alias_expr = SqlStringExpression(parent_data_set.sql_select_node.select_columns[0].column_alias)
+
+        # Build aggregate columns, labeled "min" & "max"
+        select_columns = [
+            SqlSelectColumn(
+                expr=SqlAggregateFunctionExpression(
+                    sql_function=SqlFunction.from_aggregation_type(aggregation_type=agg_type),
+                    sql_function_args=[parent_column_alias_expr],
+                ),
+                column_alias=agg_type.value,
```

Comment:
`min` and `max` are reserved keywords in a lot of dialects, aren't they? They're also kind of nondescript. I wonder if we want to do `min_<name>` and `max_<name>` instead.

Reply:
This is how it's currently implemented in MFS for the Tableau integration (with just `min` and `max` as column names). Seems to be working as expected. I wanted to make sure SLG would be able to know what the column name would be to select it - but maybe in SLG we can do a `startswith` check on the column name instead to see if it starts with `min_` or `max_`. Let me sync with @aiguofer on this.

Reply:
Diego's on board. @tlento would it make more sense to do `<name>__min` and `<name>__max` to follow our usual dunder pattern? Not sure where we're at in terms of getting rid of dunders these days...

Reply:
I implemented this with `min_<name>` and `max_<name>` for now - LMK if you think that should change!